code/__DEFINES/roguetown.dm

```diff
@@ -273,12 +273,6 @@ GLOBAL_LIST_EMPTY(job_respawn_delays)
 #define TRIUMPH_CAT_MISC "MISC!"
 #define TRIUMPH_CAT_ACTIVE_DATUMS "ACTIVE"
 
-
-#define TRIUMPH_CAT_ROUND_EFX "ROUND-EFX"
```

Comment:
I assume this was what was causing errors?

Reply:
Duplicate defines. Kicks a minor yellow error because it's trying to define the same thing twice. It was just annoying to see every time I compiled.